scripts/sonic_sku_create.py

```diff
@@ -0,0 +1,457 @@
+#! /usr/bin/env python2
+"""
+usage: sonic_sku_create.py [-h] [-v] [-f FILE] [-m [MINIGRAPH_FILE]] [-b BASE]
+                           [-r] [-c [{new_sku_only,l2_mode_only}]] [-k HWSKU]
+                           [-p] [-vv]
+Create a new SKU
+
+optional arguments:
+  -h, --help            show this help message and exit
+  -v, --version         show program's version number and exit
+  -f FILE, --file FILE  SKU definition from xml file. -f OR -m must be provided when creating a new SKU
+  -m [MINIGRAPH_FILE], --minigraph_file [MINIGRAPH_FILE]
+                        SKU definition from minigraph file. -f OR -m must be provided when creating a new SKU
+  -b BASE, --base BASE  SKU base definition
+  -r, --remove          Remove SKU folder
+  -c [{new_sku_only,l2_mode_only}], --cmd [{new_sku_only,l2_mode_only}]
```

Comment:
Removed the l2_mode_only option, since it is deprecated. So, removed the new_sku_only mode, because it becomes the default mode. Addressed in next commit.